src/main/java/org/folio/linked/data/mapper/dto/monograph/common/CategoryMapperUnit.java

```diff
@@ -35,18 +36,25 @@ public abstract class CategoryMapperUnit implements SingleResourceMapperUnit {
 
   private final CoreMapper coreMapper;
   private final HashService hashService;
-  private final BiConsumer<Category, Object> categoryConsumer;
```

Comment:
Removed this `BiConsumer` and changed it to an abstract method `addToDestination`